### PR TITLE
add new status for ODLM

### PR DESCRIPTION
**Pull request by sgrube**
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/57339

### CODE REVIEW

File: controllers/resources/resourcestatus.go [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+	reqLogger.Info("New statuses", "statuses", statuses)
+	return
+}
+
+func getAllRouteStatus(ctx context.Context, k8sClient client.Client, names []string, namespace string) (statuses []v1alpha1.ManagedResourceStatus) {
+	reqLogger := log.WithValues("func", "getAllRouteStatus", "namespace", namespace)
+	for _, name := range names {
+		r := &routev1.Route{}
+		nsn := types.NamespacedName{Name: name, Namespace: namespace}
+		_ = k8sClient.Get(ctx, nsn, r)
```
Comment (rwhundley) [inline controllers/resources/resourcestatus.go:152]: You don't need this call anymore since it's happening in getRouteStatus now.

**Review by rwhundley — CHANGES_REQUESTED** [review]

**Review by rwhundley — APPROVED** [review]
/lgtm

**Review by tthorpe2 — APPROVED** [review]
/lgtm

File: controllers/resources/resourcestatus.go (hunk 1 above)
Comment (sgrube) [reply controllers/resources/resourcestatus.go]: fixed

**Pull request merged**